### PR TITLE
Staging

### PR DESCRIPTION
**Pull request by justinengland**
taking CATs live

### CODE REVIEW

**Review by pmaslana — APPROVED** [review]

**Review by AmineKhaldi — APPROVED** [review]

File: CNAME [hunk 1]
```diff
@@ -0,0 +1 @@
+staging.chialisp.com
```
Comment (Starttoaster) [inline CNAME:1]: :thinking:

File: docusaurus.config.js [hunk 2]
```diff
@@ -1,12 +1,12 @@
...
   title: 'Chialisp',
   tagline: 'Developing with Chialisp',
-  url: 'https://chialisp.com',
+  url: 'https://staging.chialisp.com',
   baseUrl: '/',
   onBrokenLinks: 'throw',
   favicon: 'img/favicon.ico',
   organizationName: 'Chia-Network', // Usually your GitHub org/user name.
-  projectName: 'chialisp-web', // Usually your repo name.
+  projectName: 'staging-chialisp-web', // Usually your repo name.
```
Comment (Starttoaster) [inline docusaurus.config.js:9]: :thinking:

File: docusaurus.config.js [hunk 3]
```diff
@@ -37,7 +37,7 @@ module.exports = {
           position: 'left'
         },
         {
-          href: 'https://github.com/Chia-Network/chialisp-web',
+          href: 'https://github.com/Chia-Network/staging-chialisp-web',
```
Comment (Starttoaster) [inline docusaurus.config.js:40]: :thinking:

File: docusaurus.config.js [hunk 4]
```diff
@@ -106,7 +106,7 @@ module.exports = {
           sidebarPath: require.resolve('./sidebars.js'),
           // Please change this to your repo.
           editUrl:
-            'https://github.com/Chia-Network/chialisp-web/edit/main/',
+            'https://github.com/Chia-Network/staging-chialisp-web/edit/main/',
```
Comment (Starttoaster) [inline docusaurus.config.js:109]: :thinking:

**Review by nirajpathak13 — APPROVED** [review]

**Pull request merged**